integration-cli/docker_cli_build_test.go

```diff
@@ -1246,6 +1246,33 @@ func TestBuildCopyWildcardNoFind(t *testing.T) {
 	logDone("build - copy wild card no find")
 }
 
+func TestBuildCopyWildcardInName(t *testing.T) {
+	name := "testcopywildcardinname"
+	defer deleteImages(name)
+	ctx, err := fakeContext(`FROM busybox
+	COPY *.txt /tmp/
+	RUN [ "$(cat /tmp/\*.txt)" = 'hi there' ]
+	`, map[string]string{"*.txt": "hi there"})
+
+	if err != nil {
+		// Normally we would do t.Fatal(err) here but given that
```

Comment:
I'm not sure this is the best way.  This is such a weird usecase and it would seem that this test would pass if there was a regression.  Could we check the type of error returned for known filesystems that don't support `*`